docs/cri/registry.md

```diff
@@ -26,7 +26,7 @@ version = 2
 
 # The registry host has to be a domain name or IP. Port number is also
 # needed if the default HTTPS or HTTP port is not used.
-[plugins."io.containerd.grpc.v1.cri".registry.configs."gcr.io".auth]
```

Comment:
I don't think you need to change this in this file, because `plugins."io.containerd.grpc.v1.cri".registry.configs` is removed in containerd 2.0 anyway. (and `plugins."io.containerd.cri.v1.images"` is only introduced in containerd 2.0)

In this file (registry.md), I think you only need to update line 9, which you already did.

Reply:
How about to add registry config guide for version 3?